newt/builder/cmake.go

```diff
@@ -46,6 +47,19 @@ func EscapeName(name string) string {
 	return strings.Replace(name, "/", "_", -1)
 }
 
+func trimProjectPath(path string) string {
+	proj := interfaces.GetProject()
+	path = strings.TrimPrefix(path, proj.Path()+"/")
+	return path
+}
+
+func trimProjectPathSlice(elements []string) []string {
+	for e := range elements {
+		elements[e] = trimProjectPath(elements[e])
```

Comment:
I would be a bit careful with this.  In Go, slices are passed by reference, so this line is actually modifying the slice that the caller passed in.  It might be better to do one of the following:
1. Return a new slice containing the trimmed elements, or
2. Change the function type such that it doesn't return anything.

From the function type, a user might assume the function doesn't modify his original element slice.